reactjs.md

```diff
@@ -169,3 +169,21 @@ When you click a submit button, usually page gets refreshed. You can prevent thi
 ```js
 event.preventDefault();
 ```
+
+# Lists and Keys 
+
+This is the example of how we can use lists and keys in react. 
+We loop through the numbers array using the JavaScript map() function. We return a `<li>` element for each item. Finally, we assign the resulting array of elements to listItems
```

Comment:
The list may not be iterating always on a numbers array. Ideally, it should be written as an Array of Objects. 

And <li> is not a mandatory it can be <div> as well. Can you please add more generic way